cmd/avalanche-network-runner/control/control.go

```diff
@@ -489,7 +502,13 @@ func restartNodeFunc(cmd *cobra.Command, args []string) error {
 	defer cli.Close()
 
 	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
-	info, err := cli.RestartNode(ctx, nodeName, client.WithExecPath(avalancheGoBinPath), client.WithWhitelistedSubnets(whitelistedSubnets))
+	info, err := cli.RestartNode(
+		ctx,
+		nodeName,
+		client.WithExecPath(avalancheGoBinPath),
+		client.WithWhitelistedSubnets(whitelistedSubnets),
+		client.WithNodeLogLevel(nodeLogLevel),
```

Comment:
It's a bit weird seeing only the Node Log level here but no node configs?

Reply:
There was a separate PR for node configs. The treatment of global/individual node configs among start, addnode, restartnode is not so uniform. I think shall be part of a new, different PR.

Reply:
OK. Please create a ticket for that.